docs/resources/privatelink_aws.md

```diff
@@ -94,3 +117,67 @@ This resource depends on CloudAMQP instance identifier, `cloudamqp_instance.inst
 `cloudamqp_privatelink_aws` can be imported using CloudAMQP internal identifier.
 
 `terraform import cloudamqp_privatelink_aws.privatelink <id>`
+
+## Create PrivateLink with additional firewall rules
+
+To create a PrivateLink configuration with additional firewall rules, it's required to chain the [cloudamqp_security_firewall](https://registry.terraform.io/providers/cloudamqp/cloudamqp/latest/docs/resources/security_firewall)
+resource to avoid parallel conflicting resource calls. This is done by adding dependency in the firewall resource to the PrivateLink resource, `cloudamqp_privatelink_aws.privatelink`.
```

Comment:
```suggestion
resource to avoid parallel conflicting resource calls. You can do this by making the firewall resource depend on the PrivateLink resource, `cloudamqp_privatelink_aws.privatelink`.
```